back-end/src/database/migrations/20211110000658-create-users.js

```diff
@@ -0,0 +1,36 @@
+'use strict';
+module.exports = {
+  up: async (queryInterface, Sequelize) => {
+    await queryInterface.createTable('users', {
+      id: {
+        allowNull: false,
+        autoIncrement: true,
+        primaryKey: true,
+        type: Sequelize.INTEGER
+      },
+      name: {
+        type: Sequelize.STRING
```

Comment:
Creio que aqui, e também nas colunas de e-mail e password, seria melhor não permitirmos o registro `null`, certo?
```suggestion
        type: Sequelize.STRING,
        allowNull: false
```